src/injected/main.cpp

```diff
@@ -39,22 +39,26 @@ BOOL WINAPI ctrl_handler(DWORD ctrl_type)
 
 void attach_stdout(DWORD pid)
 {
-    //if (std::getenv("OL_DEBUG"))
+    //size_t requiredSize;
+    //getenv_s(&requiredSize, NULL, 0, "OL_DEBUG");
+    //if (requiredSize != 0)

```

Comment:
This should not be commented, right? I think I forgot to uncomment it before I merged

Reply:
What's with the naming though?

Reply:
i don't really understand the _s variant of this function, this is copied from some example

Reply:
You'll want something along the lines of 
```C++
size_t env_var_size;
getenv_s(&env_var_size, NULL, 0, "OL_DEBUG");
if (env_var_size > 0) // Means that the env var is set
```
Since you don't care about the value of the env var you just care whether it exists or not.